src/freertos_drivers/net_cc32xx/CC32xxSocket.cxx

```diff
@@ -222,7 +222,7 @@ int CC32xxSocket::bind(int socket, const struct sockaddr *address,
     CC32xxSocket *s = get_instance_from_fd(socket);
     if (s == nullptr)
     {
-        return - 1;
+        return -1;
```

Comment:
get_instacne_from_fd() sets the errno appropriately.  A clarifying comment has been added to the code.